inc/customizer.php

```diff
@@ -233,6 +233,53 @@ public static function parse_css_rules( array $rules ) {
 
 	}
 
+	/**
+	 * Compact CSS output.
+	 *
+	 * @link  https://github.com/GaryJones/Simple-PHP-CSS-Minification
+	 * @since NEXT
+	 *
+	 * @param  string $css
+	 *
+	 * @return string
+	 */
+	public static function compact_css( $css ) {
```

Comment:
@fjarrett 
>May not support some of the more complex syntaxes of CSS3. For that, use a proper parser and minifier.
I think we need some unit test before I feel comfortable merging that.